src/helpers/eventBroadcaster.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+const events = require('@constants/eventEndpoints')
+const requester = require('@helpers/requester')
+const common = require('@constants/common')
+
+exports.eventBroadcaster = async (
+	action,
+	{
+		requestBody = {},
+		headers = { internal_access_token: process.env.INTERNAL_ACCESS_TOKEN },
+		pathParams = {},
+		queryParams = {},
+	}
+) => {
+	try {
+		const endPoints = events.eventEndpoints[action]
+		await Promise.all(
+			endPoints.map((endPoint) => {
+				if (endPoint.type === common.EVENT_TYPE_API) {
+					if (endPoint.method === 'POST')
+						requester.post(endPoint.baseUrl, endPoint.route, headers, requestBody, queryParams)
```

Comment:
@priyanka-TL are we making API calls to to broad cast events? It should rather be internal event broad casting.

E.g.- https://nodejs.org/api/events.html, https://medium.com/deno-the-complete-reference/10-use-cases-of-event-emitters-in-node-js-d88c0b9bb536

Reply:
@aks30  this is same like mentoring, currently they are using events for internal api calls, now i added a function call also into that, so both api or function call will work, when call the publish api we can use this event

Reply:
@priyanka-TL I don't think the mentoring service uses events, user service might and they do API calls because they have to update the mentoring service if a user or org is updated. We don't need all that here, keep it to what we need only.